### PR TITLE
Switch Cargohold to YAML-only config

### PR DESCRIPTION
**Pull request by neongreen**
Should be rebased and fixed after https://github.com/wireapp/wire-server/pull/642 is merged.

### CODE REVIEW

File: deploy/services-demo/conf/cargohold.demo-docker.yaml [hunk 1]
```diff
@@ -11,3 +11,6 @@ aws:
 settings:
   maxTotalBytes: 27262976
   downloadLinkTTL: 300    # Seconds
+
+logLevel: Info
```
Comment (ChrisPenner) [inline deploy/services-demo/conf/cargohold.demo-docker.yaml]: I believe these will also have to be added here 😄 

Thanks for doing this; makes it a lot easier to configure on k8s.

Comment (neongreen) [reply deploy/services-demo/conf/cargohold.demo-docker.yaml]: Yeah, see https://github.com/wireapp/wire-server-deploy/pull/32

**Comment by fisx** [comment]
no more blocked by #642

File: .linting/duplicate-ids-whitelist.txt [hunk 2]
```diff
@@ -135,8 +135,8 @@ _optAws 2
 _optCassandra 2
 _optDiscoUrl 2
 _optGundeck 2
-_optLogLevel 2
-_optLogNetStrings 2
+_optLogLevel 3
```
Comment (ChrisPenner) [inline .linting/duplicate-ids-whitelist.txt]: We should probably talk about whether this is going to be a huge pain or not haha 😄

Comment (neongreen) [reply .linting/duplicate-ids-whitelist.txt]: I like the level of pain so far – it's just enough to make me want to spend time to rename All The Things (or fix the endpoint function mess, etc), but not enough to hate my life.

What we should really talk about is whether we are allowed to rename All The Things.

**Comment by neongreen** [comment]
I guess one day CI will pass on this one

**Comment by neongreen** [comment]
> failed to register layer: ApplyLayer exit status 1 stdout:  stderr: write /root/.stack/snapshots/x86_64-linux/custom-wire-1.1-G6NVtOVxqIB2/8.4.3/doc/hxt-9.3.1.16/doc-index.json: no space left on device

_sigh_

**Pull request merged**